src/main.js

```diff
@@ -2,7 +2,19 @@ function convertItemsToUnorderedList(listItems) {
   const ulElement = document.createElement("ul");
   listItems.forEach(item => {
     const liElement = document.createElement("li");
-    liElement.textContent = item.name;
+    liElement.classList.add("fs-api-entry");
+    if (item.type === "file") {
+      liElement.classList.add("fs-api-file");
+    } else {
+      liElement.classList.add("fs-api-directory");
+    }
+    const spanElement = document.createElement("span");
```

Comment:
Can you please rename `spanElement` to something more semantic (e.g. `nameElement`)?

We favor variable names describing the content of each variable, instead of their technical aspects.